react-native-libraries.json

```
@@ -9661,5 +9661,12 @@
     "examples": ["https://github.com/numandev1/react-native-keys/tree/master/example"],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/Sharcoux/slider",
+    "examples": ["https://codesandbox.io/s/sharcoux-slider-demo-7fqnk8?file=/src/App.js"],
```

Hey @Sharcoux, thank you for adding a new library to the directory! 👍 

One small nit before we can merge it in - for the scoped packages, we need to specify the npm package name too.
```suggestion
    "githubUrl": "https://github.com/Sharcoux/slider",
    "npmPkg": "@react-native-assets/slider",
    "examples": ["https://codesandbox.io/s/sharcoux-slider-demo-7fqnk8?file=/src/App.js"],
```